src/test/resources/tax-category-key-1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "44e5e889-0878-4c30-941f-767cebebeb7f",
+  "version": 18,
+  "name": "Standard tax category",
+  "key": "standard_tax_category_1",
+  "rates": [
+    {
+      "name": "5% US",
+      "amount": 0.051,
+      "includedInPrice": true,
+      "country": "US",
+      "id": "ak80m7Jk",
+      "subRates": []
```

Comment:
Can you add subrates as well?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/130/commits/b67ea810704559e6059bb98c99e854811ce2bdc5